### PR TITLE
chore(infocols): design review

### PR DESCRIPTION
**Pull request by kishore03109**
Resolves [ISOM-965](https://linear.app/ogp/issue/ISOM-965/infocol-design-review)

### CODE REVIEW

**Comment by kishore03109** [comment]
* **#95** <a href="https://app.graphite.dev/github/pr/isomerpages/isomer-components/95?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kishore03109 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by kishore03109** [comment]
!run chromatic

File: src/templates/next/components/complex/InfoCols/InfoCols.tsx [hunk 1]
```diff
@@ -68,7 +68,7 @@ const InfoCols = ({
   const bgColor = backgroundColor === "gray" ? "bg-gray-100" : "bg-white"
   return (
     <section className={bgColor}>
-      <div className={`${ComponentContent} py-24 px-8 sm:px-16 lg:px-24`}>
+      <div className={`${ComponentContent} py-24 px-6 sm:px-16 lg:px-24`}>
```
[the remark this answers is not in the document]
Comment (dcshzj) [reply src/templates/next/components/complex/InfoCols/InfoCols.tsx]: Wait sorry I don't see it removed?

[the remark this answers is not in the document]
Comment (kishore03109) [reply src/templates/next/components/complex/InfoCols/InfoCols.tsx]: it was removed by [alex pr ](https://github.com/isomerpages/isomer-components/commit/5325453293782dd2824e69565dd2cf6dac76bb3d)

**Review by dcshzj — APPROVED** [review]

**Pull request merged**